### ISSUE TITLE
Domain availability check in request form should be consistent (requested and alternates)

### ISSUE THREAD

**Issue by Katherine-Osos** [cisagov/manage.get.gov#776]
### Current Behavior
During the application process, on the ".gov Domain" step, the form behavior is inconsistent. When entering a first-choice domain, you have to click a button to initiate the search and the page reloads. However, when you enter the alternate domains, it searches upon leaving the field and the page doesn't refresh (allowing you to see the availability message right away).

### Expected Behavior
The fields should behave consistently. Work with design to come up with a consistent approach.

### Steps to Reproduce
1. Enter a first-choice domain and click the search button
2. Notice the page refreshes
3. Enter an alternate domain
4. Notice the page doesn't refresh

### Issue Links
Depends on #476

**Comment by h-m-f-t** [comment]
Duplicate of #1495
